lib/cli/cliMessage.js

```diff
@@ -21,13 +21,13 @@ module.exports = class CLIMessage extends CLICommand {
         /** @type {[string, string, any?]} */
         // eslint-disable-next-line prefer-const
         let [adapter, command, message] = (args);
-        if (adapter == undefined) {
+        if (adapter == null) {
```

Comment:
is that really compatible??

Reply:
why not in general !

Reply:
yes, because checking `undefined == null` and in this contexts, you can use only one of them to check for both, but we can also use `(adapter === undefined || adapter === null)`. I would prefer it, however, I don't want to spoil the party.

Reply:
lets better check both if equivalent